pylzy/tests/scenarios/none_result/__init__.py

```diff
@@ -2,11 +2,12 @@
 
 
 @op
-def just_return_none() -> None:
+def just_return_none(a: int, b: int) -> None:
+    print(f"a={a},b={b}")
     return None
 
 
 with Lzy().workflow(name="wf", interactive=False):
     # noinspection PyNoneFunctionAssignment
-    res = just_return_none()
+    res = just_return_none(1, 1)
```

Comment:
I think just_return_none(1, 1) will use different slots for 1 and 1.
Maybe use here something like that:
```
a = foo()
res = just_return_none(a, a)
```

Reply:
No, 1,1 will also use the same slot due to integer pool in python